bioptim/gui/plot.py

```diff
@@ -217,6 +218,7 @@ def __init__(
 
         self.t = []
         self.t_integrated = []
+        self.t_to_plot = []
```

Comment:
What is the purpose of `t_to_plot`? I feel that `t` and `t_integrated` are also to plot? Or do you mean it is gonna be on the y axis? Can you find a more descriptive variable name?

Reply:
I am opened to suggestions ;)

Reply:
Not there anymore